addons/udes_stock/models/stock_move_line.py

```diff
@@ -39,6 +39,7 @@ def _prepare_result_packages(self, package, result_package, products_info):
         picking = self.mapped('picking_id')
         picking.ensure_one()
         target_storage_format = picking.picking_type_id.u_target_storage_format
+        Package = Package.with_context(move_line_id=self[0].id)
```

Comment:
The semi-arbitrary [0] here suggests that we should maybe pass move_line_ids in the context, rather than move_line_id.  The subclass's stock.move.line.new_package_name_xxx() method can then verify that the move lines are actually allowed to be placed into the same package.   For example: a policy that requires all move lines to belong to the same sale.order could use self.mapped('move_id.sale_line_id.order_id').ensure_one()